scripts/validate_oas_examples.py

```diff
@@ -28,9 +28,12 @@
 REPO_ROOT = os.path.abspath(os.path.join(SCRIPT_LOCATION, ".."))
 spec_path_json = os.path.join(REPO_ROOT, "build/e-referrals-service-api.json")
 spec_path_yaml = os.path.join(REPO_ROOT, "specification/e-referrals-service-api.yaml")
```

Comment:
Did you find we need to use both json and yaml in this script?

Reply:
Yes this script uses both JSON and YAML files. It parses YAML file to get JSON response and request file path using the ref keyword. It may be avoidable. But it can be done via tech debt.